packages/augur-ui/src/modules/common/form.styles.less

```diff
@@ -2462,6 +2462,18 @@ div.error > div > div > div > div > input {
         border: @size-1 solid var(--color-primary-text);
       }
     }
+
+    &:hover:not(.loading) {
```

Comment:
Consider adding `cursor: pointer` to that hover. The cursor only changes when there is something in the category. When it's 0, even though it's clickable, the cursor visual is not the clickable one.